### PR TITLE
CB-15203 fix Swagger UI of Environment, FreeIPA, Datalake, Redbeams, Core and Periscope/Autoscale service

### PR DESCRIPTION
**Pull request by biharitomi**
The `springfox-boot-starter` package needed to be imported instead of the two legacy package: `springfox-swagger2` and `springfox-swagger-ui`. The redirect URL for the APIDocControllers needed to be fixed.

I have tested the following API doc endpoints:
 - http://localhost:8088/environmentservice/apidocs
 - http://localhost:9091/cb/swagger-ui/
 - http://localhost:8090/freeipa/apidocs
 - http://localhost:8087/redbeams/apidocs
 - http://localhost:8086/dl/apidocs
 - http://localhost:8085/as/apidocs

I added the `x-cdp-actor-crn` header for these calls.
[Based on the official documentation of SpringFox](https://springfox.github.io/springfox/docs/current/#release) the `springfox-core` lib is also eliminated.

### CODE REVIEW

**Review by doktoric — APPROVED** [review]

**Review by horadla23 — APPROVED** [review]

File: environment/build.gradle [hunk 1]
```diff
@@ -77,13 +77,10 @@ dependencies {
...
-    implementation     group: "io.springfox",              name: "springfox-swagger2",                       version: swagger2Version
-    implementation     group: "io.springfox",              name: "springfox-core",                           version: swagger2Version
-    implementation     group: "io.springfox",              name: "springfox-swagger-ui",                     version: swagger2Version
     implementation     group: "com.google.code.gson",      name: "gson",                                     version: gsonVersion
     implementation     group: "dnsjava",                   name: "dnsjava",                                  version: dnsjavaVersion
-    implementation     group: "io.springfox",              name: "springfox-swagger2",                       version: swagger2Version
-    implementation     group: "io.springfox",              name: "springfox-swagger-ui",                     version: swagger2Version
+    implementation     group: "io.springfox",              name: "springfox-core",                           version: swaggerSpringFoxVersion
```
Comment (lajosrodek) [inline environment/build.gradle:82]: Sanity check: is `springfox-core` still needed?

**Review by lajosrodek — APPROVED** [review]
LGTM, just one q

File: environment/build.gradle (hunk 1 above)
Comment (biharitomi) [reply environment/build.gradle:82]: I haven't found it as a dependency of `sprinfox-boot-starter` in maven central, so haven't tried to remove it

**Comment by lajosrodek** [comment]
Thanks for the efforts of double-checking and removing `springfox-core`! 😃

**Pull request merged**